src/Sdk/DTGenerated/Generated/TaskAgentHttpClientBase.cs

```diff
@@ -483,6 +485,16 @@ public TaskAgentHttpClientBase(Uri baseUrl, HttpMessageHandler pipeline, bool di
                 queryParams.Add("runnerVersion", runnerVersion);
             }
 
+            if (os != null)
+            {
+                queryParams.Add("os", os);
```

Comment:
make sure you double check configure the runner with your change with github.com
I am 99.9% sure the extra query param will just no-opt on the service side.

Reply:
done